ssl/pqueue.c

```diff
@@ -17,7 +17,7 @@ struct pqueue_st {
 
 pitem *pitem_new(unsigned char *prio64be, void *data)
 {
-    pitem *item = OPENSSL_malloc(sizeof(*item));
+    pitem *item = OPENSSL_zalloc(sizeof(*item));
```

Comment:
Why is this necessary?
Could you please clarify what uninitialised read is occurring?